src/hexo/themes/documentation/layout/partials/styles.hbs

```diff
@@ -0,0 +1,17 @@
+<link rel="stylesheet" href="{{url_for "/core/css/about.css"}}">
+<link rel="stylesheet" href="{{url_for "/core/css/base.css "}}">
+<link rel="stylesheet" href="{{url_for "/core/css/color.css "}}">
+<link rel="stylesheet" href="{{url_for "/core/css/footer.css "}}">
+<link rel="stylesheet" href="{{url_for "/core/css/helpers.css "}}">
+<link rel="stylesheet" href="{{url_for "/core/css/layouts.css "}}">
+<link rel="stylesheet" href="{{url_for "/core/css/media.css "}}">
+<link rel="stylesheet" href="{{url_for "/core/css/navigation.css "}}">
+<link rel="stylesheet" href="{{url_for "/core/css/search-results.css "}}">
+<link rel="stylesheet" href="{{url_for "/core/css/structure.css "}}">
+<link rel="stylesheet" href="{{url_for "/core/css/type.css "}}">
+<link rel="stylesheet" href="{{url_for "/core/css/highlight.css "}}">
+<link rel="stylesheet" href="{{url_for "/components/breadcrumb/breadcrumb.css "}}">
+<link rel="stylesheet" href="{{url_for "/components/pagination/pagination.css "}}">
+<link rel="stylesheet" href="{{url_for "/core/css/controls.css "}}">
+<link href="https://fonts.googleapis.com/css?family=Montserrat:200,300,400,500,600" rel="stylesheet">
```

Comment:
Do we need all those sizes?

Reply:
@alrra I have no idea. I just moved things from one place to another. @ststimac might know more but probably worth opening an issue

Reply:
[Done](https://github.com/sonarwhal/sonarwhal.com/issues/218).

Reply:
@alrra I think we are using all of these, but will double check.